apostrophecms/apostrophe
# PRO-2254 No attachments builder

## Purpose
Modifies `attachments` query builder to add a launder. REST requests can ask no attachments through query builders.
Doesn't show batch operations on relationShip modals.

## How to test
Try to select all pieces (with a large amount), it should not be as long as it was (I noticed the busy loader is blocked but didn't take time to check why).
Check that on relationship modals we don't have batch operations anymore
